jx_sqlite/aggs_table.py

```diff
@@ -398,22 +398,46 @@ def _get(row):
                         )
             else:  # STANDARD AGGREGATES
                 for details in s.value.to_sql(schema):
-                    for sql_type, sql in details.sql.items():
-                        column_number = len(outer_selects)
-                        sql = sql_aggs[s.aggregate] + "(" + sql + ")"
-                        if s.default != None:
-                            sql = "COALESCE(" + sql + ", " + quote_value(s.default) + ")"
-                        outer_selects.append(sql + " AS " + _make_column_name(column_number))
-                        index_to_column[column_number] = ColumnMapping(
-                            push_name=s.name,
-                            push_column_name=s.name,
-                            push_column=si,
-                            push_child=".",  # join_field(split_field(details.name)[1::]),
-                            pull=get_column(column_number),
-                            sql=sql,
-                            column_alias=_make_column_name(column_number),
-                            type=sql_type_to_json_type[sql_type]
-                        )
+                    if len(query.edges)==0 and len(query.groupby)==0 and s.aggregate=="count":
```

Comment:
This is not good, it is a special case designed to pass the test.

`test_count` was created to catch a complication with aggregates over multiple types.  Well, there is only one aggregate that makes sense over multiple types, and that is `count`.  The other aggregates operate on numbers, so do not have this same problems (but they might, should we ever decide to distinguish between `double` and `integer`).  

How do you count property values scattered over multiple database columns (string, number, boolean, object, nested)?  We definitely must `count` values in each column, then we can add those sums for a final result. It is more complicated when dealing with objects and nested object arrays; this will require storing more information in sqlite so the query does not get too complicated:

The ElasticSearch solution is to record the primitive properties values as leaves; as path/value pairs, just like `jx-sqlite` is doing now.  In addition, the ES translator is recording existence: either object existence, or nested array existence.  This existence indicator simplifies two troubling scenarios:

1. The inner object has no properties - `{}` is a valid document, it does not collapse to `null` like `[]` does.  Without any primitive objects to store we need some way to record that this empty object exists (and we need tests for this too).
2. SQL to determine if a nested property exists is complicated - It would be nice to perform queries on the fact table, like `{"exists":"a.b"}` that will return true if `a.b` is nested, but false if it does not exist. An existence indicator will make the SQL simpler by avoiding the join required to determine if there are any nested documents.